recipes/libx264/all/conanfile.py

```diff
@@ -60,11 +62,15 @@ def _configure_autotools(self):
                 args.append('--enable-shared')
             else:
                 args.append('--enable-static')
-            if self.settings.os != 'Windows' and self.options.fPIC:
+            if self.settings.os != 'Windows' and self.options.get_safe("fPIC"):
                 args.append('--enable-pic')
             if self.settings.build_type == 'Debug':
                 args.append('--enable-debug')
             args.append('--bit-depth=%s' % str(self.options.bit_depth))
+            if self.settings.os == "Macos" and self.settings.arch == "armv8":
```

Comment:
```suggestion
            if tools.is_apple_os(self.settings.os) and self.settings.arch == "armv8":
```

Reply:
sorry, can't test other configurations right now, it's out of the scope of PR

Reply:
Change this to 
```suggestion
            if self.settings.compiler == "apple-clang" and self.settings.arch == "armv8":
```
?

Reply:
The CCI team just wants to add the profile by default not actually have it pass every where... given the scope of the work we can circle back to these